src/main/java/org/generationcp/middleware/dao/GermplasmDAO.java

```diff
@@ -1549,7 +1549,9 @@ private SQLQuery addScalarsForGermplasmDTO(final SQLQuery sqlQuery) {
 
 	public long countGermplasmMatches(final GermplasmMatchRequestDto germplasmMatchRequestDto) {
 		final StringBuilder queryBuilder =
-			new StringBuilder(" select count(1) from germplsm g ");
+			new StringBuilder(" select count(1) from germplsm g "
```

Comment:
Can you add the PUI query only if there are germplasm PUIs in the germplasm match request?